src/parse.cpp

```diff
@@ -1,5 +1,19 @@
-#include "parser.h"
+#include "./lib/Lexer.h"
+#include "./lib/Parser.h"
 /* Write a main function in src/parse.cpp that uses the lexer to turn the input into tokens, then uses the parser to create an AST out of the tokens.
 After constructing the tree, print it in infix form (keep parentheses in place), and then print the value.
 If there is a parse error, print an error message and return with exit code 2. Your message should read: “Unexpected Token at Line L Column C: Token”
 */
+
+
+int main() {
+    Lexer lex(std::cin);
+    // USED JUST FOR TESTING
+    // lex.print();
+
```

Comment:
Maybe call functions more in here instead of having one function that does everything?
- allows for better reusability


```suggestion
Parser parse(lex.getTokens); //creates AST and checks for errors
std:: cout << parse.print() << '\n'; //prints in infix notation
std:: cout <<parse.calculate() << '\n'; // prints end result
delete parse; // deletes AST and cleans memory
```